### ISSUE TITLE
Details of the hyperparameters

### ISSUE THREAD

**Issue by jiamings** [ChenyangSi/FreeU#3]
Thanks for this interesting work. While I was trying to implement this, I find that some key hyperparameters are missing, for example:
 - What are the radius and radius threshold for the skip features?
 - Why use only half of the channels for (x)?
 - What are default recommended values for s and b? Are they the same for different models?

Thanks!

**Comment by haofanwang** [comment]
Agree. The idea looks simple, but more details are needed. When will the code be released?

**Comment by ChenyangSi** [comment]
We will release the code this week.

**Comment by jfischoff** [comment]
r_thresh is the main one I would like to be able to repro

**Comment by changjinhan** [comment]
I'm curious about that too.

**Comment by ChenyangSi** [comment]
We have released the FreeU code.

**Comment by jfischoff** [comment]
Thank you!

**Comment by jfischoff** [comment]
Quick question about r_thresh. It is `1` in the code. Where any other values useful?

**Comment by lifeofstudy** [comment]
> Quick question about r_thresh. It is `1` in the code. Where any other values useful?

I wonder too
